README.md

```diff
@@ -77,7 +74,28 @@ With your dependencies set up, you can run Metaphysics by running:
 yarn dev
 ```
 
-Which will start the server on http://localhost:5001
+This will pull the needed environment variables into .env.shared. If you need to override any of these values
```

Comment:
I think this is fine to explain, but this is a confusing location because the command above (`yarn dev`) doesn't actually do what this describes.